index.js

```diff
@@ -142,8 +146,6 @@ walletRouter.use((req, res, next) => {
     return;
   }
 
-  console.log('Received request to', req.originalUrl)
```

Comment:
Why don't we change this to a `logger.debug`? I think it's helpful to log the requests the wallet headless is receiving.

Reply:
I removed this log because we are using `morgan` to log all requests.